### PR TITLE
Add new docs section for DNS Analytics

### PR DESCRIPTION
**Pull request by ggalmazor**
In this PR:
- New section for the DNS Analytics endpoint

image

### CODE REVIEW

**Review by jacegu — COMMENTED** [review]
I think we need to clarify some bits before moving forward.

File: content/v2/dns-analytics.markdown [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+<info>
+Applying groupings to the query will add columns to the response payload.
+</info>
+
+### Filters
+
+Name | Default value                   | Description
+-----|---------------------------------|------------
+`:start_date` | The day before the current date | Only include results starting from the provided date
+`:end_date` | 31 days before the current date | Only include results up to the provided date
```
[the remark this answers is not in the document]
Comment (ggalmazor) [reply content/v2/dns-analytics.markdown:48]: Indeed. I'll fix this!

File: content/v2/dns-analytics.markdown [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+The default sorting policy is by ascending `date`, then ascending `zone_name`.
+
+### Pagination
+
+Pagination is supported in this endpoint:
+
+- The default page size (`per_page` query parameter) is 1,000
+- The maximum supported page size (`per_page` query parameter) is 10,000
+- Please refer to the [main guide](/v2/#pagination)
```
[the remark this answers is not in the document]
Comment (ggalmazor) [reply content/v2/dns-analytics.markdown:76]: Good idea! I'll do it

[the remark this answers is not in the document]
Comment (ggalmazor) [reply content/v2/dns-analytics.markdown:76]: See https://github.com/dnsimple/dnsimple-developer/pull/557/commits/ac965dfa66cc302dfefec38f2a3916a89a158552

File: content/v2/dns-analytics.markdown [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+### Filters
+
+Name | Default value                   | Description
+-----|---------------------------------|------------
+`:start_date` | The day before the current date | Only include results starting from the provided date
+`:end_date` | 31 days before the current date | Only include results up to the provided date
+
+<info>
+Both filters **must be used together**, and date values must be in **ISO 8601 format** (e.g. `2024-08-01`). Both dates are **inclusive**.
```
[the remark this answers is not in the document]
Comment (ggalmazor) [reply content/v2/dns-analytics.markdown]: See https://github.com/dnsimple/dnsimple-developer/pull/557/commits/ac965dfa66cc302dfefec38f2a3916a89a158552

I finally opted to remove the whole sentence while we discuss this.

File: content/v2/dns-analytics.markdown (hunk 1 above)
[the remark this answers is not in the document]
Comment (ggalmazor) [reply content/v2/dns-analytics.markdown]: See https://github.com/dnsimple/dnsimple-developer/pull/557/commits/ac965dfa66cc302dfefec38f2a3916a89a158552

File: content/v2/dns-analytics.markdown [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+<info>
+Applying groupings to the query will add columns to the response payload.
+</info>
+
+### Filters
+
+Name | Default value                   | Description
+-----|---------------------------------|------------
+`:start_date` | 31 days before the current date | Only include results up to the provided date
```
Comment (jacegu) [inline content/v2/dns-analytics.markdown:47]:
```suggestion
`:start_date` | 31 days before the current date | Only include results starting from the provided date
```

File: content/v2/dns-analytics.markdown [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+`:end_date` | The day before the current date | Only include results starting from the provided date
```
Comment (jacegu) [inline content/v2/dns-analytics.markdown:48]:
```suggestion
`:end_date` | The day before the current date | Only include results up to the provided date
```

**Review by jacegu — APPROVED** [review]
LGTM. I believe thi is ready for @itsalyse's eyes.

**Review by itsalyse — APPROVED** [review]

**Pull request merged**